plugins/GSdx/GSFunctionMap.h

```diff
@@ -125,7 +125,7 @@ template<class KEY, class VALUE> class GSFunctionMap
 			KEY key = i->first;
 			ActivePtr* p = i->second;
 
-			if(p->frames > 0)
+			if( p->frames && ttpf )
```

Comment:
Yes GSdx doesn't use this convention. That why I didn't ask you to change it. I know ref change a couple of them. And personally I might have change some others too (in particular in my code ;)).